Scripts/Linux/InstallFromSource.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# To test on a Debian-based Linux machine:
+#
+# ./InstallFromSource.sh
+# (outputs Git, GCM, and Scalar versions)
+#
+# Small clone should work:
+# scalar clone https://dev.azure.com/gvfs/ci/_git/ForTests
+#
+# Big clone for authenticated repo should work:
+# scalar clone <url>
+#
+# Note: if you are using a headless Linux machine, then GCM Core will
+# not be able to store your credentials. Instead, create a PAT and
+# use that for password (multiple times). This will be fixed soon.
+
+# Nuke previous data
+rm -rf git-vfs.deb gcm-core scalar log.txt
```

Comment:
I found I needed `git-vfs*.deb` here to really clean up the artifacts like `git-vfs_2.28.0.vfs.0.0.deb`.